packages/api/db/migration/20240112220609_create_animal_type_tables.js

```diff
@@ -0,0 +1,75 @@
+/*
+ *  Copyright (c) 2024 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+export const up = async function (knex) {
+  await knex.schema.createTable('default_animal_type', (table) => {
+    table.increments('id').primary();
+    table.string('type_key').defaultTo(null);
```

Comment:
I think this column could be the just named `key`, and should this actually be not nullable?

Reply:
Definitely should not be nullable! I wonder if keeping `breed_key` and `type_key` like in the ERD will make it a little easier to search the codebase down the line, but since we're going for non-prefixing with `id` I guess we should go all in 😄